pkg/client/uniregistrar/client.go

```diff
@@ -47,7 +48,7 @@ func (c *Client) CreateDID(driverURL string, opts ...CreateDIDOption) (string, [
 	jobID := uuid.New().String()
 
 	reqBytes, err := json.Marshal(didmethodoperation.RegisterDIDRequest{JobID: jobID,
-		AddPublicKeys: createDIDOpts.publicKeys, Options: createDIDOpts.options})
+		AddPublicKeys: createDIDOpts.publicKeys, Options: createDIDOpts.options, AddServices: createDIDOpts.services})
```

Comment:
I see the field names are coming from trustbloc-did, but `Add` prefix in the field name looks bit weird.

Reply:
@rolsonquadras  https://github.com/decentralized-identity/universal-registrar/blob/master/swagger/api.yml#L93